admin-portal/src/components/navbar/navbar.tsx

```diff
@@ -1,10 +1,66 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+
+import { Hooks, useAuthContext } from "@asgardeo/auth-react";
 
 import "./navbar.css";
+import { useLocation } from "react-router-dom";
 
 interface NavBarProps {}
 
 export function NavBar(params: NavBarProps) {
+  const { state, signIn, signOut, on, trySignInSilently } = useAuthContext();
+
+  // const search = useLocation().search;
+  // const stateParam = new URLSearchParams(search).get("state");
+  // const errorDescParam = new URLSearchParams(search).get("error_description");
+
+  // useEffect(() => {
+  //   if (stateParam && errorDescParam) {
+  //     if (errorDescParam === "End User denied the logout request") {
+  //       setHasLogoutFailureError(true);
+  //     }
+  //   }
+  // }, [stateParam, errorDescParam]);
```

Comment:
Shall we remove this?

Reply:
need these for the login error handling scenario